packages/core/src/components/tag/_common.scss

```diff
@@ -11,14 +11,16 @@ $tag-margin: ($pt-input-height - $tag-height) / 2 !default;
 $tag-height-large: $pt-grid-size * 3 !default;
 $tag-padding-large: ($tag-height-large - $pt-icon-size-large) / 2 !default;
 
+$tag-removable-selector: "&.pt-tag-removable";
```

Comment:
no need for variable here cuz it can't be customized anymore. just do the normal selector thing and write it out in css below.